src/api/PluginManager.ts

```diff
@@ -43,28 +53,30 @@ export async function startPlugins() {
                 zip.closeEntry();
 
                 if (manifest.name in plugins) throw new Error(`Plugin ${manifest.name} already registered`);
-                if (!isPluginEnabled(manifest.name)) continue;
+                if (!isPluginEnabled(manifest.name)) {
+                    plugins[manifest.name] = new Plugin(manifest);
+                    const settingsPlugins = window.Aliucord.settings.get("plugins", {});
+                    settingsPlugins[manifest.name] = false;
```

Comment:
why

Reply:
> why

spaghetti to have enabled/disabled plugins list